### DISCUSSION TITLE
Alternative for removed file gateway from version 1

### DISCUSSION THREAD

**Discussion by Shoekrates** [terminal42/contao-notification_center#373] [Q&A]
Hi,
in version 1 I used the file gateway to add the form data to a csv-file and sent it along with a second notification.
The file gateway was dropped in version 2 – so is there any replacement/alternative to create a csv file from the form data and send it?
Contao 5.3, PHP8.3

**Comment by Toflar** [comment]
No, there's no replacement at the moment. It's also mentioned in the upgrade docs.

**Reply by 9teufel** [reply]
My use case: Save Isotope orders as XML files via the Notification Center and import them into the ERP system.
A file gateway would be a great Pro feature. 👍🏻
